alf/utils/math_ops.py

```diff
@@ -182,3 +182,19 @@ def argmin(x):
     i = torch.cumsum(num_mins, 0)
     i = torch.cat([torch.tensor([0]), i[:-1]])
     return c[i]
+
+
+def shuffle(values):
+    """Shuffle a nest.
+
+    Shuffle all the tensors in `values` by a same random order.
```

Comment:
Now single quote will only emphasize the string. If we want to render a code string, then use double quotes. 

\``value\``